build/main.go

```diff
@@ -247,6 +253,8 @@ func main() {
 		buildGengen()
 	case "generate-genesis":
 		generateGenesis()
+	case "build-migrations":
+		buildMigrations()
```

Comment:
Please roll this into `build` so that migrations are always built. Similarly I expect their tests to be always run.

Reply:
✔️